GUI/src/main/kotlin/ui/components/general/FileSelector.kt

```diff
@@ -0,0 +1,31 @@
+package ui.components.general
```

Comment:
As this is not a Composable, should it reside in the components folder?

Reply:
I considered Components more general than just Composables, so imo it is not directly wrong, but we should probably have a seperate folder for uitility or so